### DISCUSSION TITLE
v0.13.1

### DISCUSSION THREAD

**Discussion by thebaer** [writefreely/writefreely#486] [Releases]
## User-Facing Changes

* Fix title rendering as list item (#470)
* Remove additional slash in `/feed/` redirect
* Don't include image alt text in post `description` metadata (#306)
* Fix SQLite-backed instances locking up (#467)
* Fix backslashes added to list items in Classic editor (#480)

## Admin-Facing Changes

* Return error when generic OAuth provider doesn't return a user ID, and prompt admin to add `map_user_id` (#469)
* Fix "Collection.hostName is empty!" log on Stats page (#468)

## Upgrading from v0.13.0

Download the latest release for your operating system. Stop running your `writefreely` service, replace all files in your installation with the ones in the archive, and then start your `writefreely` service again.

No additional steps are needed to upgrade from **v0.13.0**. Follow the instructions in each [previous release](https://blog.writefreely.org/tag:release) if you're upgrading from an earlier version.

## Developers

If you are running a custom build: we made changes to UI components that need to be compiled. Run this to compile everything:

```
make ui
```

## Contributors

Thanks to everyone who contributed bug reports, code, and code review to this release!

@stayinside
@bandicootdancing
@PeterCxy
@DaarkMoon
@poVoq
@Tealk
@thebaer
@mkyos

<hr /><em>This discussion was created from the release <a href='https://github.com/writefreely/writefreely/releases/tag/v0.13.1'>v0.13.1</a>.</em>

**Comment by poVoq** [comment]
Just upgraded, seems to work now. Thanks.

**Reply by thebaer** [reply]
Great! Glad to hear it.
